buildScripts/publish-private.sh

```diff
@@ -19,15 +20,22 @@ if [ -z "$ADO_ARTIFACTORY_API_KEY" ]; then
 fi
 
 export PUBLISHER=$ADO_ARTIFACTORY_DEVELOPER-private
-export GIT_HEAD=`git rev-parse --short HEAD`
+GIT_HEAD=$(git rev-parse --short HEAD)
+export GIT_HEAD
 
 cp vss-extension.json vss-extension-private.json
 
-npx tfx extension unshare -t $ADO_ARTIFACTORY_API_KEY --extension-id jfrog-azure-devops-extension --publisher $PUBLISHER --unshare-with $ADO_ARTIFACTORY_DEVELOPER 2>/dev/null
-npx tfx extension unpublish -t $ADO_ARTIFACTORY_API_KEY --extension-id jfrog-azure-devops-extension --publisher $PUBLISHER
-npx tfx extension create --manifest-globs vss-extension-private.json --publisher $PUBLISHER
-npx tfx extension publish -t $ADO_ARTIFACTORY_API_KEY --publisher $PUBLISHER --manifests vss-extension-private.json --override "{\"public\": false, \"version\": \"$RANDOM.$RANDOM.$RANDOM\", \"description\": \"Commit SHA: $GIT_HEAD\"}" --share-with $ADO_ARTIFACTORY_DEVELOPER
-npx tfx extension install --publisher $PUBLISHER --extension-id jfrog-azure-devops-extension --service-url https://$ADO_ARTIFACTORY_DEVELOPER.visualstudio.com -t $ADO_ARTIFACTORY_API_KEY
+npx tfx extension unshare -t "$ADO_ARTIFACTORY_API_KEY" --extension-id jfrog-azure-devops-extension --publisher "$PUBLISHER" --unshare-with "$ADO_ARTIFACTORY_DEVELOPER" 2>/dev/null
+npx tfx extension unpublish -t "$ADO_ARTIFACTORY_API_KEY" --extension-id jfrog-azure-devops-extension --publisher "$PUBLISHER"
+npx tfx extension create --manifest-globs vss-extension-private.json --publisher "$PUBLISHER"
+# Check that vsix size is less then 30MB
+vsixSize="$(du -m -- *.vsix | awk '{print $1}')"
```

Comment:
Ensure that only the first one tooked
```suggestion
vsixSize="$(du -m -- *.vsix | awk '{print $1}' | head -1)"
```